### PR TITLE
Added case result to zaakdetail api view

### PR DESCRIPTION
**Pull request by damm89**
Related to https://github.com/GemeenteUtrecht/ZGW/issues/702

### CODE REVIEW

File: backend/src/zac/core/api/serializers.py [hunk 1]
```diff
@@ -198,6 +198,7 @@ class Meta:
             "vertrouwelijkheidaanduiding",
             "deadline",
             "deadline_progress",
+            "resultaat",
```
Comment (sergei-maertens) [inline backend/src/zac/core/api/serializers.py:201]: This will only include the `resultaat` URL reference, no? You'll need a nested serializer for the `resultaat`, as evidenced by the missing test where it's not `None` as well :-)

**Review by sergei-maertens — CHANGES_REQUESTED** [review]

**Review by sergei-maertens — APPROVED** [review]
